packages/matchbox/src/components/RadioCard/RadioCard.tsx

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { RadioButtonChecked, RadioButtonUnchecked } from '@sparkpost/matchbox-icons';
 import { Box } from '../Box';
 import Group from './Group';
 import { StyledLabel, StyledInput, StyledUnchecked, StyledChecked, StyledHeader } from './styles';
 
-const RadioCard = React.forwardRef(function RadioCard(props, userRef) {
+export type RadioCardProps = {
+  checked?: boolean;
+  'data-id'?: string;
+  'data-track'?: string;
+  defaultChecked?: boolean;
+  disabled?: boolean;
```

Comment:
Shouldn't a few of these be provided by `React.ComponentPropsWithoutRef<'input'>`?